scripts/deploy.sh

```diff
@@ -26,9 +26,11 @@ ILK_ENCODED=$(seth --to-bytes32 "$(seth --from-ascii ${ILK})")
 # build it
 dapp --use solc:0.5.12 build
 
+[[ -z "$NAME" ]] && NAME="RWA-001";
+[[ -z "$SYMBOL" ]] && SYMBOL="RWA001";
```

Comment:
Would it be more appropriate to simply error out if these are undefined? I may be missing some context on how this is intended to be used, but seems like it could lead to mistakes to just silently deploy with these defaults if a user forgets to  set the values explicitly.